docs/packages/pkg/AllTheFlopsThreads.rst

```diff
@@ -0,0 +1,28 @@
+.. spelling::
+
+    AllTheFlopsThreads
+
+.. index:: unsorted ; AllTheFlopsThreads
+
+.. _pkg.AllTheFlopsThreads:
+
+AllTheFlopsThreads
+==================
+
+.. warning::
+
+    This page is a template and contains no real information.
+    Please send pull request with real description.
+
+-  `__FIXME__ Official <https://__FIXME__>`__
+-  `__FIXME__ Hunterized <https://github.com/hunter-packages/__FIXME__>`__
+-  `__FIXME__ Example <https://github.com/ruslo/hunter/blob/master/examples/__FIXME__/CMakeLists.txt>`__
+-  Available since `__FIXME__ vX.Y.Z <https://github.com/ruslo/hunter/releases/tag/vX.Y.Z>`__
```

Comment:
It will be tricky to maintain this on practice. I expect this part contributed by pull request but at time when pull request is opened we don't know when package will be released exactly... I vote for not using this in new packages, user can deduce Hunter version from related pull request (see line below).

Reply:
I personally like the "Available since" tag. It is MUCH easier for the user this way. The documentation can be updated after the release. By putting `..` before the line the whole line can be made into a comment and can be updated afterwards

Reply:
> It is MUCH easier for the user this way. The documentation can be updated after the release.

I understand that but I don't know how to keep it consistent on practice.

Reply:
yes, thats tricky. But not more tricky than keeping the wiki consistent. I think it is the same level of trickyness

I could update the template to have the "Available since" line added as comment. Then it would still be there, but not hindering contributions

Reply:
> But not more tricky than keeping the wiki consistent. I think it is the same level of trickyness

Different. Before this change I was asking developers to create/update `pkg.foo` wiki page **after** merge done and I've created release, now `pkg/foo.rst` is a **part of pull request** and release version is not known for sure. User have to create second pull request with "Available since".

Reply:
yes, you are right. This work will most probably be on your part (or MAYBE the original contributor) :-/

Reply:
I've added Hunter badge to "Available since", example:
* https://docs.hunter.sh/en/latest/packages/pkg/ArrayFire.html

However I still not sure about usefulness of this feature. I see few scenarios:
* User already has URL/SHA1 in project and want to check if it's possible to add new `hunter_add_package(foo)`. He goes to `pkg.foo` and see that this package is not available for Hunter version he is currently using. He can use version from `Available since X.Y.Z` **but** it's better to update to **latest** version of Hunter since there may be new stable versions of `foo` there.
* Even if `Available since` satisfied the build of project may end with error. In this case user should use **latest** version of Hunter (at least try to confirm)
* Instead of going to `pkg.foo` he can just add `hunter_add_package(foo)` and if package is not available Hunter will report clear error about it ("project not found"). He can go and update to **latest** version of Hunter.
* User starting project from scratch and want to use project `foo`. He should start with **latest** Hunter version and there is no need to check `Available since` part.

> edit: updated genindex view

After [updating requirements.txt](https://github.com/ruslo/hunter/commit/4ee10f9b0bf1f89a25fe1abb6d8fce7e64df92cf) this page looks different:
* https://docs.hunter.sh/en/latest/genindex.html

Reply:
yes, it's always best to update to the newest version. But it may also be useful to know how far back one can go to still have project `foo` available. I think it is not mandatory, but nice to have

yes, it looks different. A more flat style. I liked the old style better, the "subsections" were visibly more clearly.
Was there a reason for upgrade? did the style change in this version-jump? Should we somehow make our own theme for the `genindex` page?

Reply:
> I liked the old style better

Me too, may be there is a way to use old style?

> Was there a reason for upgrade?

Having internal errors after badge code added I made an update and it fixed everything.

> Should we somehow make our own theme for the genindex page?

Reply:
> Me too, may be there is a way to use old style?

maybe if the you keep `-sphinx-rtd-theme==0.1.9` in `requirements.txt`

> Should we somehow make our own theme for the genindex page?

I hoped you had more experience with theming. I was introduced to Shinx through your Hunter-Project :) And I am a C++ programmer (like you, I suspect) and don't have much clue about CSS, HTML etc

Reply:
> maybe if the you keep -sphinx-rtd-theme==0.1.9 in requirements.txt

Well probably, but I don't want to move back. We will have to use new version if there will be other improvements/features introduced.

> I hoped you had more experience with theming

Nope. Only trivial copy/paste + substitute :)

Reply:
then I think we should open an Issue and hope maybe someone dislikes the genindex page enough to fix the style ^^